docs/pages/database-access/guides/azure-sql-server-ad.mdx

```diff
@@ -62,7 +62,88 @@ login to SQL Server.
 
 ![Azure SQL Server Azure Active Directory admin page](../../../img/database-access/guides/sqlserver/azure-set-ad-admin.png)
 
-## Step 4/9. Configure Azure Managed Identities
+## Step 4/9. Configure IAM permissions for Teleport
+
+The Teleport Database Service needs Azure IAM permissions to:
+* Discover and register SQL Server instances.
+* Fetch virtual machine managed identities used for login.
+
+### Configure an Azure service principal
+
+Teleport requires the following permissions:
+* `<resource-type>/read` permissions for discovery, but only for the resource
+  types you have. For example, `Microsoft.Sql/managedInstances/read`.
+* `Microsoft.Compute/virtualMachines/read`.
+
+Here is a sample role definition allowing Teleport to read **Azure SQL Servers**
+and **Azure SQL Managed Instances**:
+
+```json
+{
+    "properties": {
+        "roleName": "SQLServerAutoDiscovery",
+        "description": "Allows Teleport to discover SQL Servers and SQL Managed Instances.",
+        "assignableScopes": [
+            "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+        ],
+        "permissions": [
+            {
+                "actions": [
+                    "Microsoft.Sql/managedInstances/read",
+                    "Microsoft.Sql/servers/read",
+                    "Microsoft.Compute/virtualMachines/read"
+                ],
+                "notActions": [],
+                "dataActions": [],
+                "notDataActions": []
+            }
+        ]
+    }
+}
+```
+
+The `assignableScopes` field above includes a subscription
+`/subscriptions/<subscription>`, allowing the role to be assigned at any
+resource scope within that subscription or the subscription scope itself.
+If you want to further limit the `assignableScopes`, you can use a resource
+group (`/subscriptions/<subscription>/resourceGroups/<group>`) or a management
+group (`/providers/Microsoft.Management/managementGroups/<group>`) instead.
+
+Now go to the [Subscriptions](https://portal.azure.com/#view/Microsoft_Azure_Billing/SubscriptionsBlade)
+page and select a subscription.
+
+Click on **Access control (IAM)** in the subscription and select **Add** >
+**Add custom role**:
+
+![IAM custom role](../../../img/database-access/guides/azure/add-custom-role@2x.png)
+
+In the custom role creation page, click the **JSON** tab and click **Edit**,
+then paste the JSON example and replace the subscription in `assignableScopes`
+with your own subscription id:
+
+![Create JSON role](../../../img/database-access/guides/sqlserver/create-role-from-json.png)
+
+## Step 5/9. Configure virtual machine identities
+
+On the **Teleport Database Service virtual machine instance** **Identity**
+section, enable the system assigned identity. This is used by Teleport to access
+Azure APIs.
+
+![System assigned identity page](../../../img/database-access/guides/sqlserver/system-managed-identity.png)
```

Comment:
Could/should the screenshot be expanded to show the nav bar and/or breadcrumbs, to help the user find/identify this section of the instance settings? Similar to the screenshot two above.